src/i18n/withTranslation.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import i18next from './config';
+import { I18nextProvider, withTranslation } from 'react-i18next';
+
+export function withTrans(WrappedComponent: any) {
+  WrappedComponent = withTranslation()(WrappedComponent);
```

Comment:
I don't think that it is necessary to have this HOC here.

https://react.i18next.com/latest/withtranslation-hoc

Did you have a specific reason for doing this?